src/UI/Components/Diagram/helpers.ts

```diff
@@ -440,3 +440,39 @@ export const findCorrespondingId = (
     attributeName,
   })).find(({ id }) => id === instanceAsTable.id);
 };
+
+/**
+ * Toggle the highlighting of a loose element in a diagram cell view.
+ * @param {dia.CellView} cellView - The cell view containing the element.
+ * @param {"add" | "remove"} kind - The action to perform, either "add" to add highlighting or "remove" to remove highlighting.
+ * @returns {void}
+ */
+export const toggleLooseElement = (
+  cellView: dia.CellView,
+  kind: "add" | "remove",
+): void => {
+  switch (kind) {
```

Comment:
when having a switch case, it is always safe practice to have a default case

Reply:
Here it's either of those, with default to no action, which would be the case here with, or without default. I don't know if my thought process is correct here.

Reply:
yes, but the default case can also simply do a break. I think, and that was also a rule at my previous company; that having the default case statement present in the switch case is best practice. The typing says it should never get there, but typing is not bulletproof.  

Adding a default case is a form of defensive programming. It helps catch unexpected or unforeseen scenarios. Even if, based on the current understanding of the code, it seems like the switch statement should cover all possible cases, future changes, refactoring, or external factors might introduce new cases.

it can just do a break, or throw an error. Whichever you choose.